src/helpers/analytics.coffee

```diff
@@ -0,0 +1,64 @@
+_ = require 'underscore'
+Router = require 'react-router'
+{HistoryLocation} = require 'react-router'
+DestinationHelper = require '../helpers/routes-and-destinations'
+
+# generate custom event data for routes
+Events =
+  viewTaskStep: ({courseId}) ->
+    # track all work done on a course
+    Analytics.sendEvent 'Course', 'Work', label: courseId
+
+  viewStudentDashboard: ({courseId}) ->
+    # compare activity between courses
+    Analytics.sendEvent 'Student', 'Dashboard', label: courseId
+
+
+# Translators convert a url like '/foo/bar/123/baz/1' into a simplified one like just '/foo/bar'
+Translators =
```

Comment:
We can just send the href.  That's the default if a Translator method doesn't exist.  We really don't need any of these, but I thought you wanted analytics to track clean urls that would remain stable even if the app changes paths around.  if that's not the case, then we can just remove all these (or maybe just leave a few?)